forage-android/src/main/java/com/joinforage/forage/android/ui/ForageElement.kt

```diff
@@ -6,10 +6,21 @@ import com.joinforage.forage.android.core.element.StatefulElementListener
 import com.joinforage.forage.android.core.element.state.ElementState
 
 /**
- * Configuration details required by ForageElement to operate correctly
+ * The configuration details that Forage needs to create a functional [ForageElement].
  *
- * @property merchantId Your merchant FNS number
- * @property sessionToken A temporary token used for authenticate
+ * Pass a [ForageConfig] instance in a call to
+ * [setForageConfig][com.joinforage.forage.android.ui.AbstractForageElement.setForageConfig] to
+ * configure an Element.
+ *
+ * @property merchantId Either a unique seven digit numeric string that
+ * [FNS](https://docs.joinforage.app/docs/ebt-online-101#food-and-nutrition-service-fns) issues
+ * to authorized EBT merchants, or a unique merchant ID that Forage provides during onboarding.
```

Comment:
Would be good to provide an example, maybe using the Kdoc `@sample` or some other means?

Reply:
@djoksimo In the last PR, I struggled with formatting issues with `@sample`. Is it okay with you if I just do them all at once at the end? 

I created a [backlog ticket](https://linear.app/joinforage/issue/TW-415/add-sample-code-tags-after-the-quickstart-has-been-approved-and-other) for this task during the last PR.